messaging/src/main/java/org/axonframework/commandhandling/gateway/IntervalRetryScheduler.java

```diff
@@ -149,36 +95,12 @@ private boolean scheduleRetry(Runnable commandDispatch, int interval) {
          *                      retry
          * @return the current Builder instance, for fluent interfacing
          */
-        public Builder retryInterval(int retryInterval) {
+        public Builder retryInterval(long retryInterval) {
```

Comment:
Small note: this changes our public API.

Reply:
Small, but important. This means it's something which isn't allowed for now, period. We should rather introduce another `retryInterval(long)` method and deprecating `retryInterval(int)`.

Reply:
This is growing bigger than it was meant to be. I 'corrected' the type, since millis are normally always kept in longs. I'll revert this, but this is the Y2K problem/bug of Unix/Linux. Strictly speaking this even should have been a `java.time.Duration`. I'll revert all `long`s in method parameters back to `int`s.